### ISSUE TITLE
Normalization for Depth

### ISSUE THREAD

**Issue by koutilya-pnvr** [lyndonzheng/Synthetic2Realistic#31]
https://github.com/lyndonzheng/Synthetic2Realistic/blob/f06db4734b89f80b76f9362ddec17f8eaf1216db/dataloader/data_loader.py#L54

The source label which is a 1 channel depth map is going through a series of transformations. Among them Normalization is one of it.  

https://github.com/lyndonzheng/Synthetic2Realistic/blob/f06db4734b89f80b76f9362ddec17f8eaf1216db/dataloader/data_loader.py#L112

But this form of Normalization only works for 3 channel images, where as depth is only 1 channel. Running this is yielding a **_Floating point exception (core dumped)_** error. Just wondering how everyone was able to run this.

BTW I am working with Virtual Kittti and Kitti datasets.

**Comment by lyndonzheng** [comment]
@koutilya40192 Thanks for pointing out this. I tried this in the latest pytorch1.2 version, we can only use one channel mean and std for the normalization. You can use transforms.Normalize((0.5, ), (0.5,)) for the depth.

**Comment by koutilya-pnvr** [comment]
Thanks for your response.
